integration-tests/test/ovmcontext.spec.ts

```diff
@@ -63,43 +45,38 @@ describe('OVM Context: Layer 2 EVM Context', () => {
   })
 
   it('Enqueue: `block.number` and `block.timestamp` have L1 values', async () => {
-    for (let i = 0; i < 5; i++) {
-      const l2Tip = await L2Provider.getBlock('latest')
-      const tx = await CanonicalTransactionChain.enqueue(
-        OVMContextStorage.address,
-        500_000,
-        '0x'
-      )
+    const tx = await env.l1Messenger.sendMessage(
```

Comment:
I made this just execute once instead of executing 5 times. I didn't see much value in having the test execute more than once. Seemed a bit arbitrary to me.

Reply:
The reason why it was done that way was to observe that it updated to the correct values over multiple blocks

Reply:
Hmmm ok I can make it 5x but will have to make some tweaks because it's too slow right now when running against kovan (takes like 5mins)